src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -1131,6 +1140,16 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
         });
     };
 
+    hideRepliesForFeedItem = (feedItemId: string) => {
+        const { feedItems } = this.state;
+        const feedItem = feedItems.find((item: FeedItem) => item.id === feedItemId);
```

Comment:
Any risk that `feedItemId` could not be contained in `feedItems`? If so, that'd result in `feedItem` would result in `undefined`. Destructoring `replies` would throw an error in this scenario

Consider the following:

```js
const { replies } = feedItems || {}
```